config.go

```diff
@@ -0,0 +1,35 @@
+package dynamodbcopy
+
+type Config struct {
+	readCapacityUnits  int64
+	writeCapacityUnits int64
+	readWorkers        int
+	writeWorkers       int
+}
+
+func NewConfig(readUnits, writeUnits, readWorkers, writeWorkers int) Config {
+	return Config{
```

Comment:
Will need to support on-demand mode which doesn't have provisioned read/write units.

Reply:
This config is just to store the parameter provided from the command line. We check for the type of billing when we fetch the current provisioning of the table, if the billing mode is `PER_REQUEST` we don't set the provisioning capacity for the table. The code then just skips the provisioning update.

Reply:
roger that